### ISSUE TITLE
Check required fields with a test [1h]

### ISSUE THREAD

**Issue by mariano-dagostino** [Gizra/drupal-starter#292]
Once https://github.com/Gizra/drupal-starter/pull/291 is merged, create some test to check required fields using drupal_test_assertions

**Comment by amitaibu** [comment]
Duplicate of https://github.com/Gizra/drupal-starter/issues/289
